ci/ci_acceptance.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+set -e
+
+echo "Building Logstash artifacts"
+rake artifact:all
+
+echo "Acceptance: Installing dependencies"
+cd qa
+bundle install
+
+echo "Acceptance: Running the tests"
+bundle exec rake test:setup
+bundle exec rake test:ssh_config
+bundle exec rake test:acceptance:all
```

Comment:
good addition, but would be awesome we could have this script parametrized by platform type, this is necessary in order to simplify the process in CI environments.